domain/consensus/processes/syncmanager/blocklocator.go

```diff
@@ -65,6 +60,7 @@ func (sm *syncManager) createBlockLocator(lowHash, highHash *externalapi.DomainH
 			return nil, err
 		}
 
+		locator = append(locator, currentHash)
```

Comment:
Moving the append to here means you are not including the highHash inside the locator.

Reply:
You're right. I thought more about the lowHash and not about the highHash. do we want the highHash inside the locator though?